File: src/main/java/io/felipeandrade/gsw2/material/vanilla/LeatherMaterial.java
package io.felipeandrade.gsw2.material.vanilla;

import io.felipeandrade.gsw2.common.GSWColor;
import io.felipeandrade.gsw2.material.GSWMaterial;

public class LeatherMaterial extends GSWMaterial {

    public static final GSWMaterial MATERIAL = new LeatherMaterial();

    public LeatherMaterial() {
        super("leather", GSWColor.saddle_brown);
    }

}

File: gsw-mod/src/main/kotlin/io/felipeandrade/gsw/material/vanilla/LeatherMaterial.kt
package io.felipeandrade.gsw.material.vanilla

import io.felipeandrade.gsw.material.GSWMaterial
import net.fabricmc.fabric.api.datagen.v1.provider.FabricRecipeProvider
import net.minecraft.data.server.recipe.RecipeJsonProvider
import java.util.function.Consumer

class LeatherMaterial : GSWMaterial("leather") {
    override fun generateRecipes(provider: FabricRecipeProvider, exporter: Consumer<RecipeJsonProvider>) {

    }

    companion object {
        val MATERIAL: GSWMaterial = LeatherMaterial()
    }
}